modules/mobile/app/models/mobile/v0/user_accessible_services.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
```

Comment:
I went back and forth on how to name the new model. It was originally `AuthorizedServices` with methods `to_a` and `available_services`. That felt wrong because availalble services was just being tacked onto accessible services despite being a superset of them. I ended up on `UserAccessibleServices` with methods `available` and `authorized`. Please let me know if you have better ideas.

Reply:
Since it looks like we aren't versioning the endpoint, this is fine as long as it ends up being returned to the FE in same shape.

Reply:
Shouldn't change the shape of the data. Both authorized services and available services are just arrays. We should be able to add to them as we wish. This PR just adds new code and doesn't change the response at all. That will come in the next PR.